### DISCUSSION TITLE
Request Body NOT  FLOAT  1.0  -> INT 1 求问

### DISCUSSION THREAD

**Discussion by yangjiyue** [fastapi/fastapi#4228] [Questions] [answered]
why?client send json   {value:2.0}  ->api-> {value:2}
四级没过不好意思, 请求提 类型标注 any ,参数值2.0接收之后为int类型2 怎么可以不转换?

**Comment by simondale00** [comment, marked as answer]
Hey, @yangjiyue.

Can you share the code for your path operation, and any pydantic models used by the path operation? This should help narrow down the issue.

Best,
Simon

**Reply by yangjiyue** [reply]
hi Simon
nice to meet you
example:↓

from pydantic import BaseModel
from typing import Optional, TypeVar, Any, List

class Item(BaseModel):
    device: str
    mark: str
    node: str
    value: Any   # client send 2.0

@app.post("/foo")
async def api_down(item: Item):
    print(item.value)  # echo 2
    return item
